lib/game_box_web/live/upload_live.ex

```diff
@@ -139,36 +139,42 @@ defmodule GameBoxWeb.UploadLive do
             />
 
             <.card_content heading={game.title} />
+            <div class="flex w-full p-4">
+              <.button
+                phx-click="delete_game"
+                phx-value-game_id={game.id}
+                label="Delete Game"
+                class="w-full"
+              />
```

Comment:
Add `data-confirm={"Are you sure you want to delete #{game.title}?"}` in this button.